FirebaseAppDistributionInternal/Sources/AppDistributionApiService.swift

```diff
@@ -374,7 +373,7 @@ struct CreateFeedbackReportRequest: Codable {
       let uploadImageTask = urlSession
         .dataTask(with: request as URLRequest) { data, response, error in
           var fadError = error
-          self.handleError(httpResponse: response as! HTTPURLResponse, error: &fadError)
+          _ = self.handleError(httpResponse: response as? HTTPURLResponse, error: &fadError)
```

Comment:
Add `@discardableResult` to the function declaration to make the `_` unnecessary. Use `git grep` in the repo to see examples.